lib/game_of_life_web/templates/universe/show.html.leex

```diff
@@ -1,7 +1,20 @@
-    <div id="universe-page">
-      <h1>It's time for... the Game... of... LIFE!</h1>
+<div id="universe-page">
+  <h1>It's time for... the Game... of... LIFE!</h1>
 
-      <div class="universe">
-        <%= render_generation(@generation, @dimensions) %>
-      </div>
-    </div>
\ No newline at end of file
+  <form action="#" phx-change="update_speed">
```

Comment:
not important, but it would be nice to use the Phoenix view helpers for `form`, `label` and `input`

Reply:
Hmm right, I'll go look them up